pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -216,6 +216,52 @@ func BuildAuthNFilter(policy *authn.Policy) *http_conn.HttpFilter {
 	}
 }
 
+func isDestinationExcludedForMTLS(destService string, mtlsExcludedServices []string) bool {
```

Comment:
since this is just comparing strings, there's no need to make the name so specific. isInStringSlice would suffice and be more reusable. can also move this to util.